vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/discovery/legacy.go

```diff
@@ -71,7 +69,7 @@ func (s *legacyRootAPIHandler) handle(req *restful.Request, resp *restful.Respon
 	clientIP := utilnet.GetClientIP(req.Request)
 	apiVersions := &metav1.APIVersions{
 		ServerAddressByClientCIDRs: s.addresses.ServerAddressByClientCIDRs(clientIP),
-		Versions:                   s.apiVersions,
+		Versions:                   []string{"v1"},
```

Comment:
looks right. not sure it's worth the trouble. but feel free to create an upstream PR.